components/prism-ichigojam.js

```diff
@@ -0,0 +1,15 @@
+// according to the offical reference https://ichigojam.net/IchigoJam-BASIC-reference.pdf
+Prism.languages.ichigojam = {
```

Comment:
The only differences between IchigoJam BASIC and standard BASIC, AFAICT, lie in the `keyword` and `function` patterns. You should really extend the original `basic` component here.